### DISCUSSION TITLE
Add Delete Confirmation dialog box

### DISCUSSION THREAD

**Discussion by lcings** [DustinBrett/daedalOS#341] [Feature Requests]
Can you add a delete confirmation dialog? I have accidentally deleted files many times.

**Comment by DustinBrett** [comment]
Thanks for the suggestion. I'll think about that as an option that can be turned on. Personally I use Windows 10 without the delete dialog. Another option I will consider for the future is a Recycle Bin.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/5101e7d9-3727-4ea7-a089-718d7d596455)
